mysql/resource_ti_resource_group.go

```diff
@@ -188,7 +191,7 @@ func DeleteResourceGroup(ctx context.Context, d *schema.ResourceData, meta inter
 	if err != nil {
 		return diag.FromErr(err)
 	}
-	// TODO: check for users assigned as safety? and assert zero?
```

Comment:
This already happens at the TiDB layer - it doesn't let you delete a resource group if it is still assigned to a user.